modules/core/src/main/java/org/apache/ignite/internal/management/baseline/BaselineTaskResult.java

```diff
@@ -273,6 +286,9 @@ public boolean isBaselineAdjustInProgress() {
             remainingTimeToBaselineAdjust = in.readLong();
             baselineAdjustInProgress = in.readBoolean();
         }
+
+        if (protoVer > V2)
```

Comment:
Read is broken. You write the `enum` first, but read `boolean`.

Reply:
Corrections have been  made, the order of properties in writeExternalData() and readExternalData() is the same now.